### ISSUE TITLE
instruction manual required

### ISSUE THREAD

**Issue by david-p-s** [strukturart/Parrot#1]
I don't know how to make Parrot work on my Nokia8110.

- I have granted permission to write into sd.
- I can see the dictionaries.
- I can access them, for example spanish_es.dic
- I can add words, edit them and save them.
- **However, the predictive T9 does not suggest any of the new words to me when I type.**

Do I have to do anything special like be root, turn off predictive, restart the phone or something?

**Comment by strukturart** [comment]
If the focus is on an input field, you can longpress # change the languages you selected in the system settings. As far as I understand the behavior of KaiOS, the new words are added to the word memory at this point.

**Comment by david-p-s** [comment]
Once I have known that the dictionary is updated when #is pressed, I have
been able to find a problem.

The bug was to have an empty word at the beginning of the list. I deleted
it and the problem was solved. Once this is known, I have discovered that
adding words with special characters also breaks the functioning. For
example, adding a word like "https: //" makes all the words behind it in
the Array do not read.
